index.rst

```diff
@@ -46,6 +46,7 @@ commitment to stability, with no breaking changes across all Conan 1.X versions.
    integrations
    configuration
    howtos
+   cheatsheet
```

Comment:
not needed, entry already exists higher up.

Reply:
removed